data/web/user.php

```diff
@@ -76,7 +76,10 @@
   $_SESSION['return_to'] = $_SERVER['REQUEST_URI'];
   $username = $_SESSION['mailcow_cc_username'];
   $mailboxdata = mailbox('get', 'mailbox_details', $username);
-
+	
+  $mailcow_hostname_array = explode(":", $mailcow_hostname)
+  $mailcow_hostname = $mailcow_hostname_array[0]
+	  

```

Comment:
That would work, but change it to use the same `strtok` solution that we have in the line below.

Reply:
This wouldn't work, to be correct. The missing semicolons `;` would cause a parse error.

Reply:
ah sure! sorry I was in python :D